Changes

```diff
@@ -1,5 +1,26 @@
 Release history of Zonemaster
 
+v2022.2.1 2023-01-31 (public fix version)
+
+ [Fixes]
+ - Restores Basci00 specification (#1123)
```

Comment:
“Basci00” -> “Basic00”

Reply:
Fixed. I think the typo comes from the PR head line...